AmazonChimeSDKDemo/AmazonChimeSDKDemo/LiveTranscriptionOptionsViewController.swift

```diff
@@ -0,0 +1,202 @@
+//
+//  LiveTranscriptionOptionsViewController.swift
+//  AmazonChimeSDKDemo
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import AmazonChimeSDK
+import UIKit
+
+class LiveTranscriptionOptionsViewController: UIViewController, UITextFieldDelegate {
+    var model: MeetingModel?
+    var meetingId = ""
+    var engineSelected = ""
+    var languageSelected = ""
+    var regionSelected = ""
+    
+    @IBOutlet var startTranscriptionButton: UIButton!
+    @IBOutlet var engineTextField: UITextField!
+    @IBOutlet var languageTextField: UITextField!
+    @IBOutlet var regionTextField: UITextField!
+    
+    func textField(_ textField: UITextField, shouldChangeCharactersIn range: NSRange, replacementString string: String) -> Bool {
+        return false
+    }
+    
+    var engines: [String] = []
+    var languages: [String] = []
+    var regions: [String] = []
+    
+    let transcribeMedicalLanguage = ["en-US"]
+    let transcribeMedicalRegions = ["ap-southeast-2", "ca-central-1", "eu-west-1", "us-east-1", "us-east-2", "us-west-2"]
+    
+    var enginesDict = [
+        "transcribe": "Amazon Transcribe",
+        "transcribe_medical": "Amazon Transcribe Medical"
+    ]
+    
+    var languagesDict = [
+        "en-US": "US English (en-US)",
+        "es-US": "US Spanish (es-US)",
+        "en-GB": "British English (en-GB)",
+        "en-AU": "Australian English (en-AU)",
+        "fr-CA": "Canadian French (fr-CA)",
+        "fr-FR": "French (fr-FR)",
+        "it-IT": "Italian (it-IT)",
+        "de-DE": "German (de-DE)",
+        "pt-BR": "Brazilian Portuguese (pt-BR)",
+        "ja-JP": "Japanese (ja-JP)",
+        "ko-KR": "Korean (ko-KR)",
+        "zh-CN": "Mandarin Chinese - Mainland (zh-CN)"
+    ]
+    
+    let regionsDict = [
+        "ap-northeast-1": "Japan (Tokyo)",
+        "ap-northeast-2": "South Korea (Seoul)",
+        "ap-southeast-2": "Australia (Sydney)",
+        "ca-central-1": "Canada",
+        "eu-central-1": "Germany (Frankfurt)",
+        "eu-west-1": "Ireland",
+        "eu-west-2": "United Kingdom (London)",
+        "sa-east-1": "Brazil (São Paulo)",
+        "us-east-1": "United States (N. Virginia)",
+        "us-east-2": "United States (Ohio)",
+        "us-west-2": "United States (Oregon)"
+    ]
+    
+    var enginePickerView = UIPickerView()
+    var languagePickerView = UIPickerView()
+    var regionPickerView = UIPickerView()
+    
+    private func notify(msg: String) {
+        DispatchQueue.main.async {
+            self.view?.makeToast(msg, duration: 2.0, position: .top)
+        }
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.engines = Array(enginesDict.keys)
+        self.languages = Array(languagesDict.keys)
+        self.regions = Array(regionsDict.keys)
```

Comment:
Looks like you are using `languagesDict` for both mapping language codes to human readable strings, and maintaining a list of language codes supported by Transcribe. This only works if `languageDict` and `regionDict` are supersets of Transcribe medical's supported languages/regions.